### PR TITLE
typos, language

### PR DESCRIPTION
**Pull request by LindsayHill**
Various typos & language updates.

Will do overview updates as a separate PR

### CODE REVIEW

File: docs/source/solutions/ipfabric/ipf_cli/advanced_cli.rst [hunk 1]
```diff
@@ -62,11 +61,11 @@ The actions in each pack have a small description on what it does.
...
 
-For more details about actions in these packs use ``st2 action get <pack-name>.<action-name>``
-command. The output can also be displayed in YAML format by appending the command with ``-y``
-flag. Similarly, for JSON use ``-j``.
+For more details about the actions in these packs, use the ``st2 action get <pack-name>.<action-name>``
+command. The output can also be displayed in YAML format by appending ``-y`` to the command.
+Similarly, for JSON use ``-j``.
 
-.. code-block:: guess
+.. code-block:: bash
```
Comment (humblearner) [inline docs/source/solutions/ipfabric/ipf_cli/advanced_cli.rst]: Ya, I have it `guess` for two reasons:

1. Build failure: https://trello.com/c/5YSnPryT/1940-sphinx-doc-sphinx-crash-when-building-botan-documentation-2413. Not sure if it is fixed yet
2. (Trivial)Guess shows some of the lines in black, whereas bash shows some in red.

Comment (LindsayHill) [inline docs/source/solutions/ipfabric/ipf_cli/advanced_cli.rst]: I was wondering about that...had just thought it was a mistake.

**Comment by LindsayHill** [comment]
Changed back from code-block bash to guess. Should fix a couple of build problems.

**Comment by emedvedev** [comment]
LGTM.

**Pull request merged**